scripts/funcotator/data_sources/getGencode.sh

```diff
@@ -158,25 +162,35 @@ function getGencodeFiles()
 
     mkdir -p ${OUT_DIR_NAME}/${refVersion}
     pushd ${OUT_DIR_NAME}/${refVersion} &> /dev/null
-    wget ftp://ftp.ebi.ac.uk/pub/databases/gencode/Gencode_human/release_${version}/gencode.v${version}.annotation.gtf.gz
-    wget ftp://ftp.ebi.ac.uk/pub/databases/gencode/Gencode_human/release_${version}/gencode.v${version}.pc_transcripts.fa.gz
 
-    gunzip gencode.v${version}.annotation.gtf.gz
-    gunzip gencode.v${version}.pc_transcripts.fa.gz
+		fileRefVersion=${version}
+		if [[ "${refVersion}" == "hg38" ]] ; then
+			sourceUrl=ftp://ftp.ebi.ac.uk/pub/databases/gencode/Gencode_human/release_${version}/gencode.v${version}.annotation.gtf.gz
+			wget ${sourceUrl} 
+    	wget ftp://ftp.ebi.ac.uk/pub/databases/gencode/Gencode_human/release_${version}/gencode.v${version}.pc_transcripts.fa.gz
```

Comment:
Your indentation is a bit off in several places -- are you mixing spaces and tabs perhaps?

Reply:
Yeah - I made changes in a few different programs that have their own settings for tabs / spaces.